<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolBar</class>
 <widget class="QWidget" name="ToolBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Show</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="saveControl">
     <property name="text">
      <string>saveControl</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="circleControl">
     <property name="text">
      <string>circleControl</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="RectangleControl">
     <property name="text">
      <string>RectangleControl</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="TriangleControl">
     <property name="text">
      <string>TriangleControl</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
